packages/build-tools/src/steps/functions/configureIosVersion.ts

```diff
@@ -64,6 +64,13 @@ export function configureIosVersionFunction(): BuildFunction {
         );
       }
 
+      if (!buildNumber && !appVersion) {
```

Comment:
Should we allow passing inputs to function groups and make the inputs available to all functions inside? Icky or necessary? (Honest question) How will `eas/build` work in a monorepo?

Reply:
TBH I'm not sure if this is necessary for v1, `eas/build` by default operates on "inputs" from `job` (if other inputs are not provided).

Reply:
>How will eas/build work in a monorepo
I hope it will work fine xD, I believe that our `eas/install_node_modules` function works well with monorepos